### PR TITLE
refactor: express waste as split.

### PR DESCRIPTION
**Pull request by n1k0**
## :wrench: Problem

Textile process `waste` is expressed as a `Ratio` while it's actually a `Split`. Also, `Unit.Ratio` as a `decodeRatio { percentage : Bool }` feature that overlaps `Split` which is responsible for typing percentages.

## :cake: Solution

Let's type waste and similar fields as `Split` and drop support for `decodeRatio { percentage : Bool }`.

## :desert_island: How to test

Ensure the app still works as expected (automated tests should do the trick).

### CODE REVIEW

File: tests/e2e-textile.json [hunk 1]
```diff
@@ -440,25 +440,25 @@
     "impacts": {
       "acd": 0.07410506399713974,
       "cch": 10.86938384256717,
-      "etf": 235.9679286920576,
-      "etf-c": 508.1155749989025,
+      "etf": 235.96792869205754,
+      "etf-c": 508.1155749989024,
```
Comment (n1k0) [inline tests/e2e-textile.json:444]: Note to reviewers: unsignificant platform-dependent rounding diffs it seems

**Review by vjousse — APPROVED** [review]

**Comment by n1k0** [comment]
@vjousse I went even further and removed support for "percentage ratio" entirely in https://github.com/MTES-MCT/ecobalyse/pull/770/commits/4c5986b2092845b1deb019377b05c3063090dbfc

File: src/Data/Impact.elm [hunk 2]
```diff
@@ -507,81 +506,6 @@ computeAggregatedScore definitions getter (Impacts impacts) =
...
-<https://www.notion.so/Rendre-param-trable-la-pond-ration-de-l-cotox-894d42e217c6448a883346203dff8db4>
-FIXME: ensure the card contents are moved to the public documentation eventually
--}
-setEcotoxWeighting : Unit.Ratio -> Definitions -> Definitions
-setEcotoxWeighting (Unit.Ratio weighting) definitions =
-    let
-        defsToUpdate =
-            [ Definition.Acd
-            , Definition.Fru
-            , Definition.Fwe
```
Comment (n1k0) [inline src/Data/Impact.elm:531]: Note to reviewers: this is obsolete dead code which was only used by… unit tests.

File: src/Data/Split.elm [hunk 3]
```diff
@@ -169,16 +169,7 @@ divideBy input split =
...
-        |> Decode.andThen
-            (\result ->
-                case result of
-                    Err error ->
-                        Decode.fail error
-
-                    Ok split ->
-                        Decode.succeed split
-            )
+        |> Decode.andThen (fromFloat >> DE.fromResult)
```
Comment (vjousse) [inline src/Data/Split.elm:172]: :heart_eyes:

**Review by vjousse — APPROVED** [review]
Nice removal of tech debt, well done!

**Pull request merged**